scripts/autorelease.js

```diff
@@ -37,16 +46,33 @@ exec(`git diff --name-only HEAD^ HEAD`, async (error, stdout, stderr) => {
     );
   });
   if (!excalidrawPackageFiles.length) {
+    console.info("Skipping release as no valid diff found");
+    core.setOutput("result", "Skipping release as no valid diff found");
     process.exit(0);
   }
 
   // update package.json
   pkg.version = `${pkg.version}-${getShortCommitHash()}`;
   pkg.name = "@excalidraw/excalidraw-next";
+  // update readme
+  let data = fs.readFileSync(`${excalidrawDir}/README_NEXT.md`, "utf8");
+
+  const isPreview = process.argv.slice(2)[0] === "preview";
+  console.info("isPreview", isPreview);
+  if (isPreview) {
+    // use pullNumber-commithash as the version for preview
+    const pullRequestNumber = process.argv.slice(3)[0];
+    pkg.version = `${pkg.version}-${pullRequestNumber}`;
```

Comment:
Is the stable `pkg.version` useful here?

I'd go with this format: `pr-${pullRequestNumber}-${getShortCommitHash()}`

Reply:
I will check if adding `pr` prefix throws semver error, if it doesn't then will go ahead with `pr.${pullRequestNumber}.${getShortCommitHash()}` as it should be in semver format else publish will throw error

Reply:
This is not possible since in [semantic version]( https://semver.org/) format `major.minor.patch` , `major`, `minor` , `patch` needs to be non negative numbers and in the `minor ` we can append string after `-` 
 in prerelease 
 https://github.com/ad1992/excalidraw/runs/5179631681?check_suite_focus=true